iati_activity_details_split_by_fields/iati_activity.py

```diff
@@ -72,17 +72,43 @@ def get_transactions_split_as_json(self):
         return [x.get_as_json() for x in self.get_transactions_split()]
 
     def _get_recipient_countries_with_normalised_percentages(self):
-        # TODO
-        return self.recipient_countries
+        """Normalise country percentages to ensure they sum to 100%"""
+        if not self.recipient_countries:
+            return []
+        total_percentage = sum(
+            country.percentage or 0 
```

Comment:
Pedantically this is vulnerable to people doing minus values, could also do `max(country.percentage, 0)` (I think, not tested) but I think at this stage we'll ignore data that that's wrong until we see signs it's an actual problem.